.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Test
```

Comment:
Let's stick with Circle for CI tests, since it's already set up for it and will integrate better into the monorepo.